src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
```

Comment:
Body don't have any paddings or borders, so I guess this rule is unnecessary here (this rule is not inherited)